apps/passport-server/src/services/generic-issuance/capabilities/FeedIssuanceCapability.ts

```diff
@@ -10,21 +19,80 @@ import { PipelineCapability } from "./types";
  * users. The server can make use of the information encoded in this Capability
  * to connect it to the other services - express routing, etc.
  */
-export interface FeedIssuanceCapability extends BasePipelineCapability {
-  type: PipelineCapability.FeedIssuance;
+export class FeedIssuanceCapability implements BasePipelineCapability {
```

Comment:
this is the main change - using a class instead of an object that satisfies the BasePipelineCapability interface